internal/service/wafv2/web_acl.go

```diff
@@ -504,14 +507,20 @@ func expandWafv2WebACLRule(m map[string]interface{}) *wafv2.Rule {
 		return nil
 	}
 
-	return &wafv2.Rule{
+	rule := &wafv2.Rule{
 		Name:             aws.String(m["name"].(string)),
 		Priority:         aws.Int64(int64(m["priority"].(int))),
 		Action:           expandWafv2RuleAction(m["action"].([]interface{})),
 		OverrideAction:   expandWafv2OverrideAction(m["override_action"].([]interface{})),
 		Statement:        expandWafv2WebACLRootStatement(m["statement"].([]interface{})),
 		VisibilityConfig: expandWafv2VisibilityConfig(m["visibility_config"].([]interface{})),
 	}
+
+	if v, ok := m["rule_label"].(*schema.Set); ok {
```

Comment:
Nit: an precautionary empty set check

```suggestion
	if v, ok := m["rule_label"].(*schema.Set); ok && v.Len() > 0 {
```